pages/howto/matlab.md

```diff
@@ -0,0 +1,498 @@
+# MATLAB on the HPC Clusters
+
+UB has a sitewide academic license for MATLAB along with all the toolboxes and CCR provides a few versions of MATLAB on our HPC Systems.
+ 
+There are a few different methods for running MATLAB on CCR HPC Systems and the method you chose depends on the type of MATLAB calculatons you with to run. This guide will focus on the main methods in which users would run MATLAB at CCR.
+These Methods include:
+
+- Running MATLAB GUI though OpenOndemand
+- Running MATLAB batch jobs on the Clusters through Slurm
+- Running MATLAB interactively on the command line
+- Using MATLAB Parallel Compute Server to submit jobs from within Matlab
+
+This guide does not give information on using MATLAB itself, it is assumed that the audience is already familiar with using it. If you are looking for specific MATLAB related info their documentation is located [here](https://www.mathworks.com/help/matlab/).
+
```

Comment:
Recommend adding a section here on how to find MATLAB versions:

## Versions of MATLAB available at CCR  

Software is updated regularly at CCR so users need to know how to search the existing software repositories to find the available versions of software they're interested in running.  In the MATLAB OnDemand app, the form provides a drop down menu listing the available versions of MATLAB.  For those using the command line, you can search using the `module spider` command.  First, specify a [software release version](../software/releases) and then search for MATLAB:  

```
module load ccrsoft/2023.01
module spider matlab

matlab:
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
     Versions:
        matlab/2021b
        matlab/2023b
```